addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -304,6 +314,41 @@ def test_systray_get_activities(self):
             }
         )
 
+    def test_multi_company_notification(self):
```

Comment:
Could you rename the test `test_cross_company_notification`, and also reposition it alphabetically in the test class ?

Reply:
Done, thanks for review :D